res/controllers/Vestax-VCI-100MKII-scripts.js

```diff
@@ -118,14 +127,44 @@ VCI102.jog = function(ch, midino, value, status, group) {
     }
 };
 
-VCI102.rate = [0, 0, 0, 0];
+VCI102.rate7bit = [64, 64, 64, 64];
+
+VCI102.rateEnable = [true, true, true, true];
 
 VCI102.rateMSB = function(ch, midino, value, status, group) {
-    VCI102.rate[ch] = (64 - value) * 128;
```

Comment:
Please add some comments to someone who looks this first time what this should do and why.